docs/cloud.rst

```diff
@@ -200,14 +178,12 @@ We can verify that the package is now present.
 
 A few caveats are in order:
 
-1. You should register the plugin before scaling to ensure that your packages
-   are installed on all the workers.
```

Comment:
Also, this caveat was removed as the `setup` method for worker plugins will run on all existing worker, as well as any new workers which show up after a `cluster.scale` call